app/routines/newflow/create_email_for_user.rb

```diff
@@ -5,7 +5,7 @@ class CreateEmailForUser
 
     protected ###############
 
-    def exec(email:, user:, is_school_issued: nil)
+    def exec(email:, user:, is_school_issued: nil, show_pin: nil)
```

Comment:
Just to be clear, this will actually pass `nil` show_pin into the method below if not specified, which might be unexpected (but then I guess `nil != false` so it'll show the pin)

Reply:
that's a good point - maybe it should default to true?

Reply:
I don't recall why I thought I should leave it ambiguous here, I agree it's a lot clearer to just default to true; pushed that change.